### ISSUE TITLE
category not choosable when popover is open

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/notes#1097]
## Steps to reproduce
1. show up popover
2. click on the category 
3. it will close immediately

![grafik](https://github.com/nextcloud/notes/assets/24530680/aaa7dfb3-1ccf-480f-864e-8940d1a5665a)

**Comment by joshtrichards** [comment]
I don't believe this is applicable any longer. Are you still experiencing this in your environment?
